website-1.x/blog/2017-12-14-introducing-docusaurus.md

```diff
@@ -5,10 +5,7 @@ authorURL: http://twitter.com/JoelMarcey
 authorImageURL: https://graph.facebook.com/611217057/picture/?height=200&width=200
 authorFBID: 611217057
 authorTwitter: JoelMarcey
-keywords:
-  - documentation
-  - blog
-  - docusaurus
+tags: [documentation, blog, docusaurus]
```

Comment:
isn't this cheating a bit? :p

Reply:
Actually docusaurus version one doesn't support this. I got the code to parse frontmater from D1 itself

Reply:
Someone made a PR for D2 and added it in blog

Reply:
ah ok so it's our new D2 parser that supports this?

Reply:
yes